### DISCUSSION TITLE
Can I get probabilities using Recbole and BERT4Rec?

### DISCUSSION THREAD

**Discussion by VladHardAndSoftWare** [RUCAIBox/RecBole#1720] [Q&A] [answered]
I know what I can get recommendation items and theirs top-k scores using recbole. But can I get propability for each recommendation items?
![image](https://user-images.githubusercontent.com/42782084/230087429-c3cbe061-52d5-47bf-9819-262862dd8de0.png)

**Comment by zhengbw0324** [comment, marked as answer]
@VladHardAndSoftWare 您好！
您可以通过sigmoid或softmax实现您的需求。
